plugins/linux-drm/meson.build

```diff
@@ -0,0 +1,11 @@
+cargs = ['-DG_LOG_DOMAIN="FuPluginLinuxDrm"']
```

Comment:
This needs a check to only load the plugin in Linux.

Reply:
I wasn't sure about the name either -- maybe linux-display?

Reply:
> This needs a check to only load the plugin in Linux.

Runtime check or build-time? We've not done the other `linux` plugins at build time, but maybe we should...

Reply:
I was thinking build time... I thought we do that already for the other linux ones but looking at it, it's **implicit**.

```
if hsi
```

Reply:
Fixed